plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -37,7 +37,18 @@ export const mergeFinalFailuresCounter = new Counter({
     help: 'Number of person merge final failures.',
 })
 
-const CASE_SENSITIVE_ILLEGAL_IDS = new Set(['[object Object]', 'NaN', 'None', 'none', 'null', '0', 'undefined'])
+const CASE_SENSITIVE_ILLEGAL_IDS = new Set([
+    '[object Object]',
+    'NaN',
+    'None',
+    'none',
+    'null',
+    '"null"',
+    '0',
+    '"0"',
+    'undefined',
+    '"undefined"',
+])
```

Comment:
don't we want to do all of these single or double quoted?

Reply:
I wondered about that... looking in the DB we only see them double quoted, but I guess that doesn't mean we'd _never_ see them single quoted 🤔